### PR TITLE
XLSX export import

### PR DESCRIPTION
**Pull request by bwrrp**
I ran across a simple XSLT library, so this adds a simple Excel export & import (implemented as update to preserve metadata). The Excel format is likely to be far more usable as input for translation than our custom json format.

### CODE REVIEW

**Comment by mvanbutselaar** [comment]
Also see the 'fdt localization messages' command, which can export to comma or tab separated files.

File: src/updateMessagesFromXlsx.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const xlsx = require('xlsx');
+
+module.exports = function updateMessagesFromXlsx (messages, xlsxPath) {
+	const workbook = xlsx.readFile(xlsxPath);
+	const firstSheet = workbook.Sheets[workbook.SheetNames[0]];
+	const rows = xlsx.utils.sheet_to_json(firstSheet);
+	const byIn = rows.reduce((byIn, row) => {
+		byIn[row.in] = row.out;
```
Comment (mvanbutselaar) [inline src/updateMessagesFromXlsx.js]: Do some sanity checking to see if there are in and out columns.

**Review by mvanbutselaar — COMMENTED** [review]
Consider explosing the import and export functions in index.js

**Comment by DrRataplan** [comment]
:shipit:

**Pull request closed without merging**